core/model/group_notification.go

```diff
@@ -0,0 +1,37 @@
+package model
+
+// GroupNotification wrapper for sending a notifications to members of a group
+type GroupNotification struct {
+	GroupID        string            `json:"group_id"`
+	Sender         *Sender           `json:"sender"`
+	Members        UserRefs          `json:"members"`
```

Comment:
It looks like `Members` could actually just be a list of user account IDs since we will be looking up their names from the group anyway. It is probably better not to ask the caller to unnecessarily specify data that we already have.

Reply:
I see your point. The user ref  is like a safety place in case we need to support external_id or another identifier along with the account id in the future. The practice shows we have such situations. I'm fine with your opinion too and could make this change. My initial direction was the same and later I decided to make it as a map. 

Could you please put your thoughts before we proceed further?